DependencyInjection/CmfSeoExtension.php

```diff
@@ -27,6 +26,9 @@
  */
 class CmfSeoExtension extends Extension
 {
+    const CLASS_SEO_METADATA_PHPCR = 'Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\SeoMetadata';
+    const CLASS_SEO_METADATA_ORM = 'Symfony\Cmf\Bundle\SeoBundle\Document\SeoMetadata';
```

Comment:
the namespace is Model, not Document. and i would call it the "default" class, not the "orm" class as its a model class. but as said below, i would not use constants in this case.